setup.py

```diff
@@ -57,6 +57,7 @@ def run_tests(self):
         'six>=1.10.0',
         'graphql-core>=0.4.9',
         'graphql-relay==0.3.3',
+        'iso8601==0.1.11',
```

Comment:
@syrusakbary OK.  I'm fine removing the version limitation there -- it's just habit for me to put it in there. I can see how in this case where you have a library used by lots of different projects that you don't want to lock down the version number.  I'll update the setup.py file.